lib/matplotlib/tests/test_lines.py

```diff
@@ -7,7 +7,7 @@
 import six
 
 import nose
-from nose.tools import assert_true, assert_raises
+from nose.tools import assert_true, assert_raises, assert_raises_regexp
```

Comment:
Im :+1:  on dropping python 2.6. I think the main reason for keeping it around is indeed RHEL6